regression-test/suites/schema_change/test_schema_change_datev2_with_delete.groovy

```diff
@@ -16,106 +16,114 @@
 // under the License.
 
 suite("test_schema_change_datev2_with_delete") {
-    sql """ SET enable_profile = true """
-    def tbName = "test_schema_change_datev2_with_delete"
-    def getJobState = { tableName ->
-         def jobStateResult = sql """  SHOW ALTER TABLE COLUMN WHERE IndexName='${tableName}' ORDER BY createtime DESC LIMIT 1 """
-         return jobStateResult[0][9]
-    }
+    // sql """ SET enable_profile = true """
```

Comment:
If the case is useless, you should remove it directly rather than comment it, and should explain that why do you need to remove these cases